zinc/src/main/scala/sbt/inc/IncrementalCompilerUtil.scala

```diff
@@ -1,17 +0,0 @@
-/*
- * Zinc - The incremental compiler for Scala.
- * Copyright 2011 - 2017, Lightbend, Inc.
- * Copyright 2008 - 2010, Mark Harrah
- * This software is released under the terms written in LICENSE.
- */
-
-package sbt
-package inc
-
-import xsbti.compile.IncrementalCompiler
-import sbt.internal.inc.IncrementalCompilerImpl
-
-object IncrementalCompilerUtil {
```

Comment:
Same comment on deprecations as I did some comments ago. I don't think we should use deprecations here. We're reworking the whole API from scratch.